extension/intents/search/search.js

```diff
@@ -359,12 +373,13 @@ intentRunner.registerIntent({
 intentRunner.registerIntent({
   name: "search.searchPage",
   async run(context) {
-    if (buildSettings.android) {
-      await performSearch(context.slots.query);
-    } else {
-      await browser.search.search({
-        query: context.slots.query,
-      });
+    await performSearch(context.slots.query);
```

Comment:
I think this will effectively revert https://github.com/mozilla/firefox-voice/commit/10672d1f754e6d426571daade72135e6fe70dff0#diff-bcfc31c6e0fe90944fc2e9703795bbb1

That is, if you change your default search provider to DuckDuckGo, and say "search for ducks" you should end up at https://duckduckgo.com/?q=ducks&t=h_&ia=news

I'm not sure what the best answer here is. Maybe just fix it for Google, and use `performSearch()` if `googleIsDefaultProvider` is true.